### PR TITLE
docs: updated aws external secret doc

### PR DESCRIPTION
**Pull request by Shubham9t9**

### CODE REVIEW

File: docs/user-guide/creating-application/secrets.md [hunk 1]
```diff
@@ -97,12 +97,49 @@ You can install `External Secrets Operator` using charts store:
...
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/secrets/add-secrets.jpg)
+
+3. Select `AWS Secret Manager` (External Secret Operator) from dropdown of `Data type`.
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/secrets/aws-secret-manager-dropdown.jpg)
+
+4. Configure secret
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/secrets/configure-aws-secret.jpg)
+
```
Comment (Abhinav-26) [inline docs/user-guide/creating-application/secrets.md:131]: add a few statements mentioning the corresponding key and property parameters in AWS Secret Store that you showed in the image below.

File: docs/user-guide/creating-application/secrets.md [hunk 2]
```diff
@@ -97,12 +97,49 @@ You can install `External Secrets Operator` using charts store:
 3. If you don't find any chart with this name i.e `external-secrets`, add chart repository using repository url  ` https://charts.external-secrets.io`. Please follow this [documentation](https://docs.devtron.ai/getting-started/global-configurations/chart-repo#add-chart-repository) for adding chart repository.
 4. Deploy the chart.
 
+#### AWS Secrets Manager
+
+To add secrets from AWS secret manager, navigate to `Secrets` of the application and follow the steps mentioned below :
+
+1. Create generic secret in the namespace using AWS access-key and secret-access-key.
```
Comment (Abhinav-26) [inline docs/user-guide/creating-application/secrets.md:104]: Add the graphical way to create a secret using the generic-helm-chart.

File: docs/user-guide/creating-application/secrets.md [hunk 3]
```diff
@@ -97,12 +97,49 @@ You can install `External Secrets Operator` using charts store:
...
+```bash
+kubectl create secret generic awssm-secret --from-file=./ access-key  \
+--from-file=./secret-access-key -n NAMESPACE
+```
+
+2. Click `Add Secret` to add a new secret.
+
+![](https://devtron-public-asset.s3.us-east-2.amazonaws.com/images/creating-application/secrets/add-secrets.jpg)
+
+3. Select `AWS Secret Manager` (External Secret Operator) from dropdown of `Data type`.
```
Comment (Abhinav-26) [inline docs/user-guide/creating-application/secrets.md:125]: Select `AWS Secret Manager` under External Secret Operator (ESO) from the dropdown of `Data type`.

**Comment by Abhinav-26** [comment]
The structure of External Secrets needs to be changed. We can have individual pages for each ESO type for eg. one page for AWS, another for Azure Vault and likewise.

File: docs/user-guide/creating-application/secrets.md (hunk 1 above)
Comment (Shubham9t9) [reply docs/user-guide/creating-application/secrets.md]: done

File: docs/user-guide/creating-application/secrets.md (hunk 2 above)
Comment (Shubham9t9) [reply docs/user-guide/creating-application/secrets.md]: done

File: docs/user-guide/creating-application/secrets.md (hunk 3 above)
Comment (Shubham9t9) [reply docs/user-guide/creating-application/secrets.md]: done

**Review by prakarsh-dt — APPROVED** [review]

**Pull request merged**